docs/examples/driver_examples/Qcodes example with Rigol DS1074 Z.ipynb

```diff
@@ -0,0 +1,317 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "_ = plot_dataset(datasaver.dataset)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, the same measurement can also be performed using a helper function `expand_setpoints_helper`. This function takes a ParameterWithSetpoints and acquires the parameter along with it's setpoints. The data is returned in a format prepared to insert into the dataset. Parameter `get_trace` is a ParameterWithSetpoints. "
```

Comment:
please rephrase this paragraph - there is no need to explain what `ParameterWithSetpoints` and `expand_setpoints_helper` are - the important part is that this particular driver implements acquiring the trace as a `ParameterWithSetpoints`, and this means that one can use the `expand_setpoints_helper` convenience function when calling `add_results`. instead of explaining what `ParameterWithSetpoints` and `expand_setpoints_helper` are in general, a link to the relevant page in the documentation should be enough.

Reply:
Thansk